README.md

```diff
@@ -47,7 +47,8 @@ import napari
 viewer = napari.view_image(data.cells3d(), channel_axis=1, ndisplay=3)
 ```
 
-![napari viewer showing an image of an astronaut.](https://github.com/napari/docs/blob/main/docs/images/screenshot-add-image.png)
+<img width="1198" alt="napari viewer showing cells in 3d." src="https://user-images.githubusercontent.com/44469195/209934762-aba0c2c0-5829-43ac-96f1-ce55707860b4.png">
```

Comment:
There does not seem to be anything else except this that link to githubusercontent, I don't have any particular preference, but I want to make sure other are Ok with that.

Reply:
The other option would be to link to the docs repo?
https://github.com/napari/docs/blob/d1c287825d065a3d7511979f97f9194d63d6da7d/docs/images/multichannel_cells.png

Reply:
that's the image used in napari.org
https://napari.org